services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:

```

Comment:
Specify inner type of `QuerySet`
```suggestion
def get_cinema_halls() -> QuerySet[CinemaHall]:
```